test/cfgexample.txt

```diff
@@ -0,0 +1,74 @@
+!
+version 15.4
+no service timestamps log datetime msec
+no service timestamps debug datetime msec
```

Comment:
Tam już jest jakiś exsmple. Bazujmy już na tym co jest

Reply:
to przypadkowo dodałem

Reply:
coś sprawdzałem i nie usunałem